doto-frontend/src/components/pages/Calendar/CalendarComponent.js

```diff
@@ -52,47 +92,85 @@ export function Content ({ children, appointmentData, style, onTaskStatusUpdated
         setOpen(false);
     };
 
-    const deleteTask = (taskId) => {
+    const deleteTask = taskId => {
         onTaskDeleted(taskId);
         document.getElementById("grid").click();
     };
 
+    const handleOpenUpdateModal = () => {
+        setOpenUpdateModal(true);
+    };
```

Comment:
There is a bug where the form for editing tasks doesn't get updated. To reproduce:
1. Click on a task. The tooltip will appear.
2. Click the edit button, edit the title and save your changes. You'll notice that the appointment tooltip is still open with the old title.
3. Click the edit button again, and you'll notice the edit form hasn't been updated.

To fix this, I think the tooltip has to be closed after clicking the `UPDATE` button. The Appointment.Tooltip component doesn't come with an out-of-the-box method of closing it. 

I suggest you copy how task deletion forcibly closes the tooltip in the `Content` function above.